examples/6field-simple/elm_6f.cxx

```diff
@@ -12,1677 +12,1682 @@
+
+  bool parallel_lagrange; // Use (semi-) Lagrangian method for parallel derivatives
+  bool parallel_project;  // Use Apar to project field-lines
+
+  //********************
+
+  Field3D Xip_x, Xip_z; // Displacement of y+1 (in cell index space)
+
+  Field3D Xim_x, Xim_z; // Displacement of y-1 (in cell index space)
+
+  bool phi_constraint; // Solver for phi using a solver constraint
+
+  BoutReal vac_lund, core_lund;     // Lundquist number S = (Tau_R / Tau_A). -ve -> infty
+  BoutReal vac_resist, core_resist; // The resistivities (just 1 / S)
+  Field3D eta;                      // Resistivity profile (1 / S)
+  bool spitzer_resist;              // Use Spitzer formula for resistivity
+
+  Field3D eta_spitzer;        // Resistivity profile (kg*m^3 / S / C^2)
+  Field3D nu_i;               // Ion collision frequency profile (1 / S)
+  Field3D nu_e;               // Electron collision frequency profile (1 / S)
+  Field3D vth_i;              // Ion Thermal Velocity profile (M / S)
+  Field3D vth_e;              // Electron Thermal Velocity profile (M / S)
+  Field3D kappa_par_i;        // Ion Thermal Conductivity profile (kg&M / S^2)
+  Field3D kappa_par_e;        // Electron Thermal Conductivity profile (kg*M / S^2)
+  Field2D omega_ci, omega_ce; // cyclotron frequency
+  Field3D kappa_perp_i; // Ion perpendicular Thermal Conductivity profile (kg&M / S^2)
+  // Electron perpendicular Thermal Conductivity profile (kg*M / S^2)
+  Field3D kappa_perp_e;
+
+  bool output_transfer; // output the results of energy transfer
+  bool output_ohm;      // output the results of the terms in Ohm's law
+  bool output_flux_par; // output the results of parallel particle and heat flux
+  // Maxwell stress, Reynolds stress, ion diamagbetic and curvature term
+  Field3D T_M, T_R, T_ID, T_C, T_G;
+  Field3D ohm_phi, ohm_hall, ohm_thermal;
+  // particle flux, ion and elelctron heat flux
+  Field3D gamma_par_i, heatf_par_i, heatf_par_e;
+
+  BoutReal hyperresist;  // Hyper-resistivity coefficient (in core only)
+  BoutReal ehyperviscos; // electron Hyper-viscosity coefficient
+  Field3D hyper_eta_x;   // Radial resistivity profile
+  Field3D hyper_eta_z;   // Toroidal resistivity profile
+
+  int damp_width;        // Width of inner damped region
+  BoutReal damp_t_const; // Timescale of damping
+
+  // Metric coefficients
+  Field2D Rxy, Bpxy, Btxy, B0, hthe;
+  Field2D I;         // Shear factor
+  BoutReal LnLambda; // ln(Lambda)
+
+  /// Ion mass
+  BoutReal Mi = SI::amu;
+
+  /// Communication objects
+  FieldGroup comms;
+
+  /// Solver for inverting Laplacian
+  std::unique_ptr<Laplacian> phiSolver{nullptr};
+  std::unique_ptr<Laplacian> aparSolver{nullptr};
+
+  /// For printing out some diagnostics first time around
+  bool first_run = true;
+
+  Field3D field_larger(const Field3D& f, const BoutReal limit) {
+    Field3D result;
+    result.allocate();
+
+    for (auto i : result) {
+      if (f[i] >= limit)
```

Comment:
warning: statement should be inside braces [readability-braces-around-statements]

```cpp
if (f[i] >= limit)
                  ^
                   {
```